src/pages/[postcode]/places/places.layout.tsx

```diff
@@ -0,0 +1,52 @@
+import { ComponentChildren } from 'preact';
+import { useTranslation } from 'react-i18next';
+import { Link, Outlet, useParams, useSearchParams } from 'react-router-dom';
+import '@etchteam/diamond-ui/control/Button/Button';
+
+import '@/components/composition/Layout/Layout';
+import '@/components/composition/PlacesHeader/PlacesHeader';
+import '@/components/content/HeaderTitle/HeaderTitle';
+import '@/components/content/Icon/Icon';
+
+export default function PlacesLayout({
+  children,
+}: {
+  readonly children?: ComponentChildren;
+}) {
+  const { t } = useTranslation();
+  const { postcode } = useParams();
+  const [searchParams] = useSearchParams();
+  const materialId = searchParams.get('materialId');
+  const materialName = searchParams.get('materialName');
+  const query = materialId
```

Comment:
What happens if we have a name but no id?

Reply:
It returns all the locations like a search never happened.

I have a separate issue for it https://linear.app/etch/issue/WRAP-368/places-search-material-not-found not sure what to do yet.

Probably needs a conditional view